strapi/strapi
# fix: Sometimes the children display order is incorrect in list view

### What does it do?
Resolves issues with inconsistent ordering of items listed in related field items.

### Why is it needed?
The `reverse` modifies `data.results` directly, instead of creating a shallow copy with revered data.

### How to test it?

1. On the `Content-Type Builder` create two collections with at least 1 of the collections having a 1toMany or ManyToMany relation field.
2. Add entries for the collection types on the `Content Manager`, linking the related field with a few entries from the other content-type.
3. On the entries list page, click on the dropdown which shows the list of related items to any particular entry.
5. Notice the consistency in the order of listed items every time you view the list of items on list page and on the entry detail page.

### Related issue(s)/PR(s)

Fixes https://github.com/strapi/strapi/issues/15676

### Demo

https://user-images.githubusercontent.com/7220167/217528688-38493ac7-df53-4c9e-9dcd-5a7599d51ba6.mov
